src/things.ts

```diff
@@ -371,32 +428,7 @@ export default class Things {
 
     try {
       const response = await fetch(
-        new URL(`${this.thingsEndpoint}/${thingId}`, this.thingsUrl).toString(),
-        options
-      )
-      if (!response.ok) {
-        const errorRes = await response.json()
-        throw this.thingError.HandleError(errorRes.message, response.status)
-      }
-      const thingData: Thing = await response.json()
-      return thingData
-    } catch (error) {
-      throw error
-    }
-  }
-
-  public async IdentifyThing (thingKey: string): Promise<Thing> {
```

Comment:
What happened to this `IdentifyThing` function. Is there a reason why we are deleting it?

Reply:
we no longer support this function on Magistrala